manager-bundle/src/Resources/skeleton/public/.htaccess

```
@@ -39,6 +39,7 @@
     #   following RewriteCond (best solution)
     RewriteCond %{ENV:REDIRECT_STATUS} ^$
     RewriteRule ^index\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]
+    RewriteRule ^app\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]
```

Isn’t there a `RewriteCond` missing?

```suggestion
    RewriteCond %{ENV:REDIRECT_STATUS} ^$
    RewriteRule ^app\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]
```

---

Can we change it to ` RewriteRule ^(index|app)\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]`?

---

@leofeyer yes, but it has to be `^(?:index|app)\.php(?:/(.*)|$)`, otherwise the `$1` would be incorrect.

---

See #6307